src/appEntry.scss

```diff
@@ -1 +1,43 @@
-@import "~bootstrap/scss/bootstrap";
\ No newline at end of file
+@import "~bootstrap/scss/bootstrap";
+
+.tableCalls{
+    tab-size: 90%;
+    color: black;
+
+}
+
+.tableHeader{
+    position: sticky; 
+    top: 0;
+}
+
+.placeHold{
+    margin-top: 95pt;
+    width: max($numbers: 1);
+    height: 180pt;
+    overflow-y: auto;
+}
+
+.button-group{
+    margin-top: 20pt;
+    margin-left: 65pt;
+}
+
+ button{
+    margin-left: 10pt;
+}
+
+.forma1{
+    margin-left: 30pt;
+    margin-bottom: 20pt;
+}
+
+.callendarSpace{
+    background-color: chartreuse;
+}
+
+.container {
+    justify-content: flex-start;
+  }
+
+.tableFixHead thead th { position: sticky; top: 0; background-color: blueviolet; }
```

Comment:
It would be nice to have some kind of [naming convention](https://en.wikipedia.org/wiki/Naming_convention_(programming)), CSS is also programing language so treat it same as any other.